packages/desktop-client/src/components/reports/reports/Custom.js

```diff
@@ -0,0 +1,344 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import * as d from 'date-fns';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+import * as monthUtils from 'loot-core/src/shared/months';
+import { amountToCurrency } from 'loot-core/src/shared/util';
+
+import useCategories from '../../../hooks/useCategories';
+import useFilters from '../../../hooks/useFilters';
+import { theme, styles } from '../../../style';
+import AlignedText from '../../common/AlignedText';
+import Block from '../../common/Block';
+import Text from '../../common/Text';
+import View from '../../common/View';
+import { AppliedFilters } from '../../filters/FiltersMenu';
+import PrivacyFilter from '../../PrivacyFilter';
+import { ChooseGraph } from '../ChooseGraph';
+import Header from '../Header';
+import { ReportOptions } from '../ReportOptions';
+import { CustomSidebar } from '../ReportSidebar';
+import { ReportLegend, ReportSummary } from '../ReportSummary';
+import { CustomTopbar } from '../ReportTopbar';
+import defaultSpreadsheet from '../spreadsheets/default-spreadsheet';
+import useReport from '../useReport';
+import { fromDateRepr } from '../util';
+
+export default function Custom() {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ naming could be improved (also applies to the filename).
```suggestion
export default function CustomReport() {
```